tempto-examples/src/main/resources/tempto-configuration-wrong.yaml

```diff
@@ -0,0 +1,84 @@
+# This configuration contains only presto and hive configured correctly
```

Comment:
Give it more meaningful name.
Genrally we want to allow only specific misconfigurations in yaml file. 
And this tests suggests that any errors are good.